src/view/Office.vue

```diff
@@ -186,6 +187,13 @@ export default {
 				return this.error
 			}
 		},
+		loadingMessage() {
+			if (this.loadingMsg) {
+				return t('richdocuments', this.loadingMsg)
```

Comment:
This won't work for translating. Either you will need to pass over an already translated message or pass over a static load message code so we can handle translations within richdocuments for each specific message.